apps/announcements/views.py

```diff
@@ -0,0 +1,33 @@
+from apps.announcements.models import Announcement
+from apps.checkin.views import success_data_jsonify
+from rest_framework.decorators import api_view
+import os
+import datetime
+from fcm_django.models import FCMDevice
+
+APPROVED_NUMBERS = []  # TODO: Add numbers here
+
+
+@api_view(['GET', 'POST'])
+def announcements(request):
+    if request.POST:
+        form = request.POST
+        body = form['Body']
+        from_number = form['From']
+        # TODO: Set the below variables
+        account_sid = form['AccountSid']
+        if from_number in APPROVED_NUMBERS and \
+                account_sid == os.environ['TWILIO_SID']:
+            a = Announcement()
+            a.message = body
+            a.time = datetime.datetime.now()
+            a.save()
+            device = FCMDevice.objects.all()
+            device.send_message(title=a.message)
```

Comment:
Don't know the firebase api super well, but I'd assume this puts it in the title field of APNS. We want it in the message field. We'll leave title as null.

Reply:
Ok, I wasn't sure. I'll do that.